### PR TITLE
Update README for w3

### PR DESCRIPTION
**Pull request by tanho63**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +33,10 @@ Don't forget to check out the [Learning Objectives for each week](https://r4ds.g
...
 
-- 1: Chapter 1, 2: Introduction, The Whole Game; Club Goals Overview
+1: Chapter 1, 2: Introduction, The Whole Game; Club Goals Overview
 
-Cohort 1 - [Jon Harmon](https://twitter.com/jonthegeek): [slides](https://r4ds.github.io/bookclub-rpkgs/Presentations/Week01/Cohort1/chapters1-2.html#1) | [video](https://youtu.be/FR6NsbkYhcw)
+- Cohort 1 - [Jon Harmon](https://twitter.com/jonthegeek): [slides](https://r4ds.github.io/bookclub-rpkgs/Presentations/Week01/Cohort1/chapters1-2.html#1) | [video](https://youtu.be/FR6NsbkYhcw)
+
+2: Chapter 3, 4: System Setup, Package Structure & State
+
+- Cohort 1 - [Asmae Toumi](https://twitter.com/asmae_toumi): slides | [video](https://youtu.be/3r-EPc9XqxE)
```
Comment (jonthegeek) [inline README.md]:
```suggestion
- Cohort 1 - [Asmae Toumi](https://twitter.com/asmae_toumi): SLIDES | [video](https://youtu.be/3r-EPc9XqxE)
```

Capitalize to call out the missing piece.

**Review by jonthegeek — CHANGES_REQUESTED** [review]
One small change.

**Review by jonthegeek — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -35,6 +33,10 @@ Don't forget to check out the [Learning Objectives for each week](https://r4ds.g
 
 ### Presentations & Recordings
 
-- 1: Chapter 1, 2: Introduction, The Whole Game; Club Goals Overview
+1: Chapter 1, 2: Introduction, The Whole Game; Club Goals Overview
 
-Cohort 1 - [Jon Harmon](https://twitter.com/jonthegeek): [slides](https://r4ds.github.io/bookclub-rpkgs/Presentations/Week01/Cohort1/chapters1-2.html#1) | [video](https://youtu.be/FR6NsbkYhcw)
+- Cohort 1 - [Jon Harmon](https://twitter.com/jonthegeek): [slides](https://r4ds.github.io/bookclub-rpkgs/Presentations/Week01/Cohort1/chapters1-2.html#1) | [video](https://youtu.be/FR6NsbkYhcw)
+
+2: Chapter 3, 4: System Setup, Package Structure & State
```
Comment (tanho63) [inline README.md:40]:
```suggestion
2: Chapter 3, 4: System Setup, Package Structure & Stat
```

**Pull request merged**